TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Prefabs/ConfigCanvas.prefab

```diff
@@ -7247,10 +7247,9 @@ MonoBehaviour:
     m_HorizontalOverflow: 0
     m_VerticalOverflow: 0
     m_LineSpacing: 1
-  m_Text: 'Increases stability of the cursor by adjusting the hand movement required
-
-    to
-    begin a movement. This stability changes during interactions.'
+  m_Text: Changes the hand movement required before the cursor begins to move. Stability
```

Comment:
Unfortunately this doesn't currently fit in the text box. I'd suggest we maybe pull the suggestions out of here and leave them for documentation and instead have only the details of what happens inside the UI.

![image](https://user-images.githubusercontent.com/56393090/124294354-7a8ea480-db4f-11eb-990e-476d24b7f1f8.png)